cmd/ocm/gcp/update-wif-config.go

```diff
@@ -197,6 +197,42 @@ func updateWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 			"or contact Red Hat support.", wifConfig.ID())
 	}
 
+	// If the WIF pool is hosted in a different project than the main project, we need to verify the pool usage
+	if wifConfig.Gcp().FederatedProjectId() != wifConfig.Gcp().ProjectId() {
+		// Check if there are associated clusters first
+		clustersAssociated, err := gcpClientWifConfigShim.HasAssociatedClusters(ctx, connection, wifConfig.ID())
+		if err != nil {
+			return errors.Wrapf(err, "failed to check for associated clusters")
```

Comment:
```suggestion
			return errors.Wrapf(err, "failed to check if wif-config '%s' has associated clusters", wif.ID())
```

Reply:
For user clarity.

Reply:
Also, this error should be actionable. In this case, the user action would be to try again or contact support.